src/Sema.zig

```diff
@@ -16103,6 +16131,7 @@ fn safetyPanic(
         .incorrect_alignment => "incorrect alignment",
         .invalid_error_code => "invalid error code",
         .index_out_of_bounds => "attempt to index out of bounds",
+        .cast_truncated_data => "integer cast truncated bits",
```

Comment:
This error is not very explanatory, because there is also the trunace inbuild https://ziglang.org/documentation/master/#truncate

Reply:
I copied the same error message as stage1. Do you have a suggestion on something else?

Reply:
should be fine then, but adding a TODO would be nice.